### ISSUE TITLE
The "quick create" thing in the planner gives too much of a search bar feel (5)

### ISSUE THREAD

**Issue by rootAvish** [openshiftio/openshift.io#3552]
### Description

In the planner board view, the quick creation bar (pictured) gives the look and feel of a search bar way too much (I know there's "add" written everywhere but you'd be surprised at the ability of people to zone out and focus only on the "work item title" thing).

Because of this looking too much like a search bar, the part that at the moment there's no search in the planner is not grokkable all too easily. I've mistakenly typed existing issue titles and pressed enter hoping for search results and instead had a new issue get created twice now. I would assume our users would have the same problem.

<img width="1359" alt="screen shot 2018-05-14 at 4 06 18 pm" src="https://user-images.githubusercontent.com/5445560/39992590-e5df5158-5790-11e8-98d6-6144008a43c5.png">

I feel the UX could be improved here.

**Comment by joshuawilson** [comment]
cc @openshiftio/uxd-team

**Comment by GeorgeActon** [comment]
@catrobson what does the (5) mean? Is that an estimate?

**Comment by catrobson** [comment]
@GeorgeActon yes that is our storypoint estimate for UXD.  Sorry, we usually do this on just UXD created stories - didn't realize we stuck it on a regular eng story!

**Comment by hrishin** [comment]
Just an input (feel free to ignore)
Can we use this as a search box and add an item as well?

e.g. Quora - its one search box. If the user is trying to search for any question it would try to search existing a questions. If not it doesn't match with existing questions it gives facility to add a question.

![quora-search-box](https://user-images.githubusercontent.com/28615674/44159792-217bb100-a0d6-11e8-9290-57a12e93c759.png)

Similar way we can do for the planner. An added advantage over here is if a user is trying to file an issue this search could give similar matches against existing issues and it could prevent filing duplicate issue.

**Comment by Veethika** [comment]
This issue has been taken care of in #4386
https://redhat.invisionapp.com/share/J2O8MP098VX
If the solution works well for the query tab it could be implemented for the Backlog as well.

**Comment by rootAvish** [comment]
@Veethika looks good, just one question here: What is the action when somebody presses "enter" after writing something in the query tab here: https://redhat.invisionapp.com/share/J2O8MP098VX#/screens/321871625 ?

**Comment by Veethika** [comment]
@rootAvish the custom query solution has been captured in this invision Doc: https://redhat.invisionapp.com/share/ESOFBERRG7U#/322048695_Cover_Page_Template

**Comment by joshuawilson** [comment]
Since this has only been designed shouldn't we leave it open until it gets changed and released to production?

**Comment by Veethika** [comment]
UX solution provided. Removing the team/UX label.

**Comment by joshuawilson** [comment]
@nimishamukherjee is there anyone working on it or is it in your backlog?
